plugins/eu.numberfour.n4js.ts.model/model/TypeRefs.xcore

```diff
@@ -578,7 +578,7 @@ abstract class StructuralTypeRef {
 	 * contains the structural members. In this case, properties 'astStructuralType' and 'genStructuralType'
 	 * are empty.
 	 */
-	refers TStructuralType structuralType
+	refers transient TStructuralType structuralType
```

Comment:
this needs to be persisted, so we can't make it transient.
(but I think all the other changes in N4JS.xcore should be ok, will run the tests with those).

THanks Moritz for the PR!